vowpalwabbit/cb_explore_adf.cc

```diff
@@ -185,12 +192,16 @@ void predict_or_learn_bag(cb_explore_adf& data, base_learner& base, v_array<exam
     else
       multiline_learn_or_predict<false>(base, examples, data.offset, i);
     assert(preds.size() == num_actions);
-    data.action_probs[preds[0].action].score += prob;
+    top_actions[preds[0].action]++;
     if (is_learn && !test_sequence)
       for (uint32_t j = 1; j < count; j++)
         multiline_learn_or_predict<true>(base, examples, data.offset, i);
   }
 
+  // generate distribution over actions
+  generate_bag(begin(top_actions), end(top_actions), begin_scores(data.action_probs), end_scores(data.action_probs));
```

Comment:
I didn't know that you can do begin(top_actions).  Is that defined in STL?  Why is this preferred to top_actions.begin()? Or is it just a different style?